test/jenkins/criu-fault.sh

```diff
@@ -39,3 +39,5 @@ fi
 ./test/zdtm.py run -t zdtm/static/fpu03 --fault 134 -f h --norst || fail
 # also check for the main thread corruption
 ./test/zdtm.py run -t zdtm/static/fpu00 --fault 134 -f h --norst || fail
+
+./test/zdtm.py run -t zdtm/static/vfork00 --fault 7 --report report -f h || fail
```

Comment:
It should likely instead be `&& fail` checking that zdtm reports an error in case of criu crash.

Reply:
Done!